paasta_tools/cli/cmds/status.py

```diff
@@ -228,8 +230,13 @@ def paasta_status_on_api_endpoint(
             service=service, instance=instance, verbose=verbose
         ).result()
     except HTTPError as exc:
-        paasta_print(exc.response.text)
+        output.append(PaastaColors.red(exc.response.text))
         return exc.status_code
+    except (BravadoConnectionError, BravadoTimeoutError) as exc:
+        output.append(
+            PaastaColors.red(f"Could not connect to API: {exc.__class__.__name__}")
+        )
+        return 1
 
```

Comment:
Can you add a big catchall `except` for everything else?
I never want a user's status to crash entirely if it can't get a status on a particular instance.

Reply:
Yep good call, done.